### ISSUE TITLE
Ethr may be busted

### ISSUE THREAD

**Issue by mfeit-internet2** [perfsonar/pscheduler#1191]
Seen in a Docker container:

```
Error from 163.253.16.3:
  Tool failed: Traceback (most recent call last):
    File "/usr/libexec/pscheduler/classes/tool/ethr/run", line 22, in <module>
      from ethr_defaults import *


Diagnostics from 163.253.16.3:
  Tool standard output:

  None

  Tool standard error:

  Traceback (most recent call last):
    File "/usr/libexec/pscheduler/classes/tool/ethr/run", line 22, in <module>
      from ethr_defaults import *
```

**Comment by mfeit-internet2** [comment]
Fixed a couple of things that prevented running.

Runs, but still fails.   Looks like it lacks a `-o` to redirect the log someplace useful.  We should capture this and include it in the diagnostics.  

```
$ pscheduler task --tool ethr throughput --dest tb-el7-prod-dr.ps.dev.internet2.edu
Submitting task...
Task URL:
https://localhost/pscheduler/tasks/909bdedf-34be-44f2-8ed2-8eb90c37c716
Running with tool 'ethr'
Fetching first run...

Next scheduled run:
https://localhost/pscheduler/tasks/909bdedf-34be-44f2-8ed2-8eb90c37c716/runs/734601b9-1b4e-41a1-9240-2d1b3d2b87de
Starts 2021-08-11T19:49:43Z (~4 seconds)
Ends   2021-08-11T19:50:08Z (~24 seconds)
Waiting for result...

Summary
Interval       Throughput     Retransmits
0.0 - 0.0      0.00bps        Not Reported

No further runs scheduled.

$ pscheduler result --diags https://localhost/pscheduler/tasks/909bdedf-34be-44f2-8ed2-8eb90c37c716/runs/734601b9-1b4e-41a1-9240-2
d1b3d2b87de | less
2021-08-11T19:49:43Z on localhost and tb-el7-prod-dr.ps.dev.internet2.edu with
  ethr:

throughput --dest tb-el7-prod-dr.ps.dev.internet2.edu

Summary
Interval       Throughput     Retransmits
0.0 - 0.0      0.00bps        Not Reported

Limit system diagnostics for this run:
  No limits were applied

Diagnostics from localhost:
  ethr -c tb-el7-prod-dr.ps.dev.internet2.edu -ports tcp=5201 -d 20s -t b -p tcp
  \Exited: 0


  Standard output:

  Unable to open the log file ethrc.log, Error: open ethrc.log: permission denieddial tcp 163.253.37.203:8888: connect: connection r
efused


  Standard error:



No additional information available from tb-el7-prod-dr.ps.dev.internet2.edu.

```

Works fine looped back onto itself, so 4.4.1 -> 4.4.1 should work okay.

```
$ pscheduler task --tool ethr throughput --dest localhost --loopback                                                              Submitting task...
Task URL:
https://localhost/pscheduler/tasks/583c9104-815d-4f54-80d5-10915b3b35b4
Running with tool 'ethr'
Fetching first run...

Next scheduled run:
https://localhost/pscheduler/tasks/583c9104-815d-4f54-80d5-10915b3b35b4/runs/70cf1a21-685a-47fb-9dab-2448f49e365d
Starts 2021-08-11T19:54:04Z (~3 seconds)
Ends   2021-08-11T19:54:29Z (~24 seconds)
Waiting for result...

* Stream ID 7
Interval       Throughput     Retransmits    Current Window
0.0 - 1.0      18.95 Gbps     Not Reported   Not Reported
1.0 - 2.0      18.59 Gbps     Not Reported   Not Reported
2.0 - 3.0      18.67 Gbps     Not Reported   Not Reported
3.0 - 4.0      17.80 Gbps     Not Reported   Not Reported
4.0 - 5.0      17.42 Gbps     Not Reported   Not Reported
5.0 - 6.0      14.27 Gbps     Not Reported   Not Reported
6.0 - 7.0      13.36 Gbps     Not Reported   Not Reported
7.0 - 8.0      15.99 Gbps     Not Reported   Not Reported
8.0 - 9.0      11.61 Gbps     Not Reported   Not Reported
9.0 - 10.0     12.04 Gbps     Not Reported   Not Reported
10.0 - 11.0    10.99 Gbps     Not Reported   Not Reported
11.0 - 12.0    13.63 Gbps     Not Reported   Not Reported
12.0 - 13.0    12.41 Gbps     Not Reported   Not Reported
13.0 - 14.0    10.54 Gbps     Not Reported   Not Reported
14.0 - 15.0    13.15 Gbps     Not Reported   Not Reported
15.0 - 16.0    13.58 Gbps     Not Reported   Not Reported
16.0 - 17.0    11.66 Gbps     Not Reported   Not Reported
17.0 - 18.0    15.58 Gbps     Not Reported   Not Reported
18.0 - 19.0    14.63 Gbps     Not Reported   Not Reported

Summary
Interval       Throughput     Retransmits
0.0 - 19.0     814.97 Mbps    Not Reported

No further runs scheduled.
```

**Comment by mfeit-internet2** [comment]
I'm happy enough with the current state of things to close this.
